src/templates/InstantSearch.js widget/.template.js

```diff
@@ -6,8 +6,15 @@ module.exports = {
   libraryName: 'instantsearch.js',
   supportedVersion: '>= 4.21.0 < 5.0.0',
   templateName: 'instantsearch.js-widget',
-  appName: 'instantsearch.js-app',
-  keywords: ['algolia', 'InstantSearch', 'Vanilla', 'instantsearch.js', 'widget'],
+  appName: 'test', // only used for test cases
```

Comment:
this should be passed in the test itself, not a default here I think

Reply:
makes sense https://github.com/algolia/create-instantsearch-app/pull/458/commits/e9095ce63be229f4badcb755a1bf2d65d7084da4